.travis.yml

```diff
@@ -130,8 +124,17 @@ matrix:
   - env: CLIENT="saucelabs:MicrosoftEdge" COMMAND=test
   - env: CLIENT=selenium:firefox:48.0 PERF=1 COMMAND=test
   - env: CLIENT=selenium:firefox:48.0 NEXT=1 COMMAND=test
-
+  - node_js: "stable"
+    services: docker
+    env: CLIENT=node COMMAND=test
   fast_finish: true
+  include:
```

Comment:
Tiny nit, could we move this to the top of the `matrix:` section, I was confused that allowed_failures had this section inside of it